@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -751,7 +696,7 @@ export class TextElementsRenderer {
                         .loadCharset(textElement.text, textElement.renderStyle)
                         .then(() => {
                             textElement.loadingState = LoadingState.Loaded;
-                            this.m_mapView.update();
+                            this.m_viewUpdateCallback();
```

Comment:
this seems unrelated, I would say next time best split into a separate commit at least